ipyleaflet/leaflet.py

```diff
@@ -2031,6 +2032,38 @@ def remove_legend_element(self, key):
         self.send_state()
 
 
+class ColormapControl(WidgetControl):
+    """ColormapControl class, with WidgetControl as parent class.
+
+    A control which contains a colormap, to be used with Choropleth.
```

Comment:
Is it really always meant to be used with a Choroplet layer?

Reply:
Not restrictively. I can be extended to any other layers requiring a colormap like for instance the velocity layer. This will be added once the ColormapControl is adopted.

Reply:
The doc string has been changed. This control can be extended to other layers possibly requiring a colormap. Once the control is adopted, it can for instance be applied to a velocity layer.